images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -14,8 +14,16 @@
   vars:
     hiveStorageType: "{{ meteringconfig_spec_overrides | json_query('storage.hive.type') }}"
 
+- name: Validate fields when OCP features are disabled
```

Comment:
Maybe we should just be checking this in a validate_reporting_operator section and remove the `when`, since if a user overrides the value to empty, in any situation (OCP or not) it's invalid.

Reply:
Yeah, makes sense to remove the `when` conditional, but I'm not sure this belongs in the `validate_reporting_operator_tls.yml` task file.

Reply:
I could either create a task file for validating this one task, or place it elsewhere. I think it makes sense to validate that this value is set for non-okd distributions early in the role execution.